### PR TITLE
Fix ScriptLoader using 0Harmony20 instead of 0Harmony (breaking change, version up to 2.0)

### PR DESCRIPTION
**Pull request by krypto5863**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,6 @@
+# Why fork?
+https://github.com/ghorsington/BepInEx.ScriptLoader/issues/4
+
 # ScriptLoader for BepInEx 5.4
```
Comment (ManlyMarco) [inline README.md]:
```suggestion
# ScriptLoader for BepInEx 5.4
```
No fork on the table

File: ScriptLoader/ScriptLoader.cs [hunk 2]
```diff
@@ -10,7 +10,7 @@
 
 namespace ScriptLoader
 {
-    [BepInPlugin("horse.coder.tools.scriptloader", "C# Script Loader", "1.2.4")]
+    [BepInPlugin("horse.coder.tools.scriptloader", "C# Script Loader", "1.2.5")]
```
Comment (ManlyMarco) [inline ScriptLoader/ScriptLoader.cs]: I prefer to not bump version numbers from PRs, also this is a breaking change so the version should probably be changed to 2.0.

File: ScriptLoader/ScriptLoader.cs [hunk 3]
```diff
@@ -10,7 +10,7 @@
 
 namespace ScriptLoader
 {
-    [BepInPlugin("horse.coder.tools.scriptloader", "C# Script Loader", "1.2.4")]
+    [BepInPlugin("horse.coder.tools.scriptloader", "C# Script Loader", "2.0")]
```
Comment (krypto5863) [inline ScriptLoader/ScriptLoader.cs:13]: Bumped it to 2.0

File: README.md (hunk 1 above)
Comment (krypto5863) [reply README.md]: Picking up utensils.

File: README.md [hunk 4]
```diff
@@ -96,4 +96,16 @@ In most cases, you can fix the script yourself. Open the script specified in the
 * Remove `using BepInEx.Harmony;` line
 * Replace `HarmonyWrapper.PatchAll` with `Harmony.CreateAndPatchAll`
 
-Then try to run the game again. If the error persists or you get some other error, the script is too complex to fix by this guide. In that case please conact the developer of the script and ask them to fix it.
\ No newline at end of file
+Then try to run the game again. If the error persists or you get some other error, the script is too complex to fix by this guide. In that case please conact the developer of the script and ask them to fix it.
+
+### Upgrading to 1.2.5.0
+
+Starting with 1.2.5.0, you might notice an error such as: `Use UnpatchSelf() to unpatch current instance...`
```
Comment (ManlyMarco) [inline README.md]:
```suggestion
### Upgrading to 2.0

Starting with 2.0, you might notice an error such as: `Use UnpatchSelf() to unpatch current instance...`
```

File: ScriptLoader/MonoCompiler.cs [hunk 5]
```diff
@@ -128,7 +128,10 @@ private static void ImportAppdomainAssemblies(Action<Assembly> import)
             {
                 if (StdLib.Contains(ass.name.Name) || compiledAssemblies.Contains(ass.name.Name))
                     continue;
-                import(ass.ass);
+                //Prevents ScriptLoader from using the old harmony library that is used for shimmed plugins.
+                if (ass.name.Name.Equals("0Harmony20"))
+	                continue;
+				import(ass.ass);
```
Comment (ManlyMarco) [inline ScriptLoader/MonoCompiler.cs]:
```suggestion
                //Prevents ScriptLoader from using the old harmony library that is used for shimmed plugins.
                if (ass.name.Name.Equals("0Harmony20"))
	                continue;
                import(ass.ass);
```

**Review by ManlyMarco — APPROVED** [review]
Looks like Horse isn't coming back, so this is the main repo now.
Changes look good, thank you for the fix!

**Pull request merged**